arangod/RestHandler/RestReplicationHandler.cpp

```diff
@@ -1923,6 +1923,11 @@ Result RestReplicationHandler::processRestoreIndexes(
         if (e.code() == TRI_ERROR_NOT_IMPLEMENTED) {
           continue;
         }
+        if (auto const& message = e.message();
```

Comment:
Could we at least check exception code here? Like `TRI_ERROR_ARANGO_DUPLICATE_NAME` or smth